### DISCUSSION TITLE
lazer osz conversion adds unnecessary inherited lines

### DISCUSSION THREAD

**Discussion by Dopaminos** [ppy/osu#21394] [Q&A] [answered]
i was recording a video for #21393 and seen, that when you opening map from lazer in stable editor, it adds unnecessary 1.0x inherited lines (green lines). i can understard that 1.8x sv for every slider is a flaw that may be fixed later (i'm using only 1.8x speed in my map and ig lazer shouldn't make a ton of same green lines). but 1.0x imo is a bug.

video with green lines and osz is attached. (rename .gif -> .osz)

https://user-images.githubusercontent.com/42616718/203591424-0c13919a-ce6c-4e2f-af91-d7797b8fc3db.mp4

map - https://user-images.githubusercontent.com/42616718/203591676-e4d57a8e-f618-42bf-9057-fb0d563c37da.gif

**Comment by peppy** [comment, marked as answer]
Thanks for reporting this. I've bumped to an issue until we have a chance to look into this further https://github.com/ppy/osu/issues/22132.
